manifests/params.pp

```diff
@@ -31,6 +31,11 @@
     $rh_certificate_consumer_host_key = undef
   }
 
+  $ruby_foreman_scap_client_package = $operatingsystem ? {
```

Comment:
You could simplify this to just `$package_name`. You can also use `$::osfamily` where `Debian` covers both Debian and Ubuntu (and their derivatives even).

Reply:
using $package_name